src/module/chat/chatrollhelpers.ts

```diff
@@ -179,14 +161,8 @@ function calculateMoveResultText(
 ): string | undefined {
   if (!move) return undefined
 
-  switch (type) {
-    case ROLL_OUTCOME.MISS:
-      return move.Miss
-    case ROLL_OUTCOME.WEAK_HIT:
-      return move.Weak
-    case ROLL_OUTCOME.STRONG_HIT:
-      return move.Strong
-  }
+  const dfOutcomeKey = DfRollOutcome[type]
+  return move[dfOutcomeKey]
```

Comment:
This change is incorrect. `move` here is an `EnhancedDataswornMove`, not a Dataforged-derived thing. Which is why Ironsworn-sheet move rolls don't display an outcome.

#422:
![CleanShot 2022-08-22 at 06 35 14](https://user-images.githubusercontent.com/39902/185934116-b15fcc7e-3c62-48db-b6da-d17e7685be11.jpg)


This patch:
![CleanShot 2022-08-22 at 06 31 31](https://user-images.githubusercontent.com/39902/185933374-c4bc7a7a-66e3-422a-9ff6-c2d9f7bc1b43.jpg)